voltdbclient/response.go

```diff
@@ -303,7 +303,8 @@ func deserializeTableForResult(r io.Reader) (rowsAff int64, err error) {
 	if err != nil {
 		return 0, err
 	}
-	if strings.Compare("modified_tuples", cname) != 0 {
+
+	if strings.Compare("modified_tuples", cname) != 0 && cname != "STATUS" {
```

Comment:
It was a bit weird that the old code assumed “modified_tuples” in the response table. The response can have any columns in the table. We shouldn’t assume anything.